src/content/changes/highlights/2021-12-03.md

```diff
@@ -0,0 +1,18 @@
+---
+date: 2021-12-03T10:00:00
+title: Highlights for the week ending December 03, 2021
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## Apps
+
+
+## User interfaces
+ - Version caching is now disabled for the selfupdate command, so that the latest version can be seen right after it’s released.
+ - Make the --release flag mandatory, which specifies the workload cluster release version in the template cluster and template nodepool subcommands.
+
+## Documentation
+ - Docs for --release flag added
```

Comment:
```suggestion

```

Reply:
I wouldn’t mention this @PrashantR31 
We usually include changes in docs if the article had significant updates or a new doc article was added.

Reply:
Sure, I've removed it.